homeassistant/components/shelly/strings.json

```diff
@@ -4,6 +4,7 @@
     "flow_title": "Shelly: {name}",
     "step": {
       "user": {
+        "description": "If you need help with set up have a look here: https://www.home-assistant.io/integrations/shelly/\n\nBefore set up, the battery-powered device must be woken up by pressing the button on the device.",
```

Comment:
We should not put urls in the description, they should be passed in as placeholders. However, we shouldn't add this here, it should be obvious that one should look at the docs for help with set up .

Reply:
If not, it should be linked from the UI for any config flow…